components/post_view/reaction/reaction.jsx

```diff
@@ -247,14 +247,19 @@ export default class Reaction extends React.PureComponent {
                 onEnter={this.loadMissingProfiles}
             >
                 <div
+                    id={`post-reaction-${this.props.post.id}-${this.props.emojiName}`}
                     className={className}
                     onClick={handleClick}
                 >
                     <span
+                        id={`${this.props.post.id}-${this.props.emojiName}-emoji`}
```

Comment:
I suggest to assign a simple ID like 'emoji' and rely on parent element to target it.
```suggestion
                        id='emoji'
```
Then get it like ``cy.get(`#postReaction-${postId}-slightly_frowning_face > #emoji`)``

Reply:
@saturninoabril I understand your point but `id='emoji'` would duplicate for each emoji so it would be wrong isn't it?. Maybe you meant something like `cy.get(`#postReaction-${postId}-slightly_frowning_face > .emoji`)` ?

Reply:
You're right.